lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -33,3 +35,71 @@ export function runRenovateRepoStats(
   }
   logger.debug({ stats: prStats }, `Renovate repository PR statistics`);
 }
+
+export interface BranchMetadata {
+  branchName: string;
+  branchSha: string | null;
+  baseBranch: string | undefined;
+  baseBranchSha: string | null;
+  automerge: boolean;
+  isModified: boolean;
+}
+
+export interface BaseBranchMetadata {
+  branchName: string;
+  sha: string;
+}
+
+export interface BranchSummary {
+  cacheModified: boolean | undefined;
+  baseBranches: BaseBranchMetadata[];
+  branches: BranchMetadata[];
+  inactiveBranches: string[];
+}
+
+export function unwrap({
```

Comment:
It's subjective, but `unwrap` function name looks misleading to me

```suggestion
export function branchCacheToMetadata({
```